.github/ISSUE_TEMPLATE/bug_template.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: 'bug, untriaged'
+assignees: ''
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Plugins**
```

Comment:
Along with Plugins we should have a component as well because we are planning to keep ansible , Helm Charts etc , all in one repository.

Reply:
We are still evaluating options but the latest thoughts is in the process of requesting a standalone opensearch-helm repo to start with.

Reply:
I will change this to component for now.